### PR TITLE
build, test: remove jsdriver which has been deprecated by IoT.js

### PR DESCRIPTION
**Pull request by yorkie**
This patch uses stable python testrunner and remove the builtin module `testdriver` from basic as well.

/cc @Rokid/nodejs

### CODE REVIEW

File: test/testsets.json [hunk 1]
```diff
@@ -106,7 +106,7 @@
     { "name": "test_timers_error.js" },
     { "name": "test_timers_simple.js", "timeout": 10 },
     { "name": "test_tls_connect_and_close.js" },
-    { "name": "test_tty.js" },
+    { "name": "test_tty.js", "skip": ["all"], "reason": "need to setup test environment" },
```
Comment (yorkie) [inline test/testsets.json]: Skip the tty because the Python TestRunner doesn't monitor the terminal, will fix it later.

**Comment by yorkie** [comment]
Just note that the upstream PR is https://github.com/Samsung/iotjs/pull/1582.

**Comment by algebrait** [comment]
Can you provide original IoT.js PR url?

**Comment by legendecas** [comment]
LGTM

**Comment by yorkie** [comment]
Merged and thank you guys for reviews :p

**Pull request merged**